ksl_alert_backend/user/userController.js

```diff
@@ -96,6 +96,7 @@ const signIn = (req, res) => {
         user
           .validatePassword(password)
           .then(match => {
+            console.log('match:', match);
```

Comment:
@JSNavey @nikhilkamineni should we remove the `console.log()` with match first? Can we do that from github?